### ISSUE TITLE
%{.install.patch} not set correctly.

### ISSUE THREAD

**Issue by plicease** [plicease/App-af#3]
See commentary : 

https://irclog.perlgeek.de/native/2017-05-11

**Comment by djerius** [comment]
Here's a test distribution: https://github.com/djerius/Alien-XPA. And the behavior:

```
% af install --dry-run
Alien::Build> error in probe (will do a share install): No hooks registered for probe at /proj/axaf/ots/pkgs/perl-5.22/x86_64-linux_debian-7/lib/site_perl/5.22.1/Alien/Build.pm line 282.
Alien::Build::Plugin::Core::Download> downloaded xpa-2.1.18.tar.gz
Use of uninitialized value in substitution iterator at /proj/axaf/ots/pkgs/perl-5.22/x86_64-linux_debian-7/lib/site_perl/5.22.1/Alien/Build/Interpolate.pm line 128.
Alien::Build::CommandSequence> + patch -p1 < /v2.1.18.diffs
sh: 1: cannot open /v2.1.18.diffs: No such file
external command failed at /proj/axaf/ots/pkgs/perl-5.22/x86_64-linux_debian-7/lib/site_perl/5.22.1/Alien/Build/CommandSequence.pm line 49.
	...propagated at /proj/axaf/ots/pkgs/perl-5.22/x86_64-linux_debian-7/lib/site_perl/5.22.1/Alien/Build.pm line 824.

```

**Comment by plicease** [comment]
This is fixed in 0.10.  Thank you for reporting this!
